web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCertificatesAvailableFunctionsImpl.java

```diff
@@ -23,16 +23,22 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import se.inera.intyg.common.lisjp.support.LisjpEntryPoint;
 import se.inera.intyg.common.support.facade.model.Certificate;
+import se.inera.intyg.infra.security.authorities.AuthoritiesHelper;
+import se.inera.intyg.infra.security.common.model.AuthoritiesConstants;
 import se.inera.intyg.webcert.web.service.facade.GetCertificatesAvailableFunctions;
 import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkDTO;
 import se.inera.intyg.webcert.web.web.controller.facade.dto.ResourceLinkTypeDTO;
 
 @Service
 public class GetCertificatesAvailableFunctionsImpl implements GetCertificatesAvailableFunctions {
 
+    @Autowired
+    AuthoritiesHelper authoritiesHelper;
```

Comment:
Because you should change to remove the null-check, you can also change the authoritiesHelper member to be final and inject it through a constructor instead. You then move the autowire-annotation to the constructor.